docs/man/man1/hal_parport.1

```diff
@@ -147,7 +147,7 @@ For each parallel port handled by the hal_parport driver, a type can optionally
 
 If the type is not specified, the default is out.
 
-A type of epp is the same as out, but the hal_parport driver requests that the port switch into EPP mode. The hal_parport driver does not use the EPP bus protocol, but on some systems EPP mode changes the electrical characteristics of the port in a way that may make some marginal hardware work better. The Gecko G540’s charge pump is known to require this on some parallel ports.
+A type of epp is the same as out, but the hal_parport driver requests that the port switch into EPP mode. The hal_parport driver does not use the EPP bus protocol, but on some systems EPP mode changes the electrical characteristics of the port in a way that may make some marginal hardware work better. The Gecko G540's charge pump is known to require this on some parallel ports.
```

Comment:
here also and so on ...
But I am neither a native english speaker nor a linguist. Maybe someone other could help out here.

Reply:
Yes, I agree, those are bad characters too.  I will either amend this PR, or start a new one to fix them as well.  Thanks!

Reply:
Sorry, I misunderstood your suggestion. 

For at least the docs I changed, my understanding is that U+0027 ( ' ) is the correct character as it is part of the Basic Latin Unicode block (encoded in one byte in UTF-8).   The look-alike characters can cause syntax errors in code and in terminal commands. 

When the English source files are converted to other languages, I am not so sure which characters are correct, that is why I left those files (_es, _fr, _CN, etc.) alone, as well as the .po files (those I left alone because I know they are being worked on).

Reply:
>For at least the docs I changed, my understanding is that U+0027 ( ' ) is the correct character as it is part of the Basic Latin Unicode block (encoded in one byte in UTF-8). 

I am not sure. You can read through the references in https://github.com/LinuxCNC/linuxcnc/pull/1677#discussion_r834481961 and see that's is not that simple.  
But for the singe and double quotes i totally agree.

>as well as the .po files (those I left alone because I know they are being worked on).

Not on these which are in the repo, but they are handled by Weblate, so totally correct to not touch them.

Reply:
i figured out that my concerns were without any reason. Asciidoc generates the correct character. (`'` turns into U+2019 )